cpp/daal/include/algorithms/k_nearest_neighbors/kdtree_knn_classification_predict_types.h

```diff
@@ -107,9 +119,70 @@ class DAAL_EXPORT Input : public classifier::prediction::Input
     services::Status check(const daal::algorithms::Parameter * parameter, int method) const DAAL_C11_OVERRIDE;
 };
 
+/**
+ * <a name="DAAL-CLASS-ALGORITHMS__RIDGE_REGRESSION__PREDICTION__RESULT"></a>
```

Comment:
DAAL-CLASS-ALGORITHMS__KDTREE_KNN_CLASSIFICATION__PREDICTION__RESULT

Reply:
Fixed, thank you